import ex_1

def test_ex1():
  pass #use assert ex_1.main()==resultado_esperado, "mensaje error..."
